src/main/java/org/commcare/formplayer/exceptions/MediaMetaDataNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package org.commcare.formplayer.exceptions;
+
+/**
+ * Throw whenever a user tries to access a media metadata that does not exist.
+ */
+public class MediaMetaDataNotFoundException extends RuntimeException {
```

Comment:
I see this exception is only used in tests currently while mocking. We should have the mocking behaviour same as the main code. So we should also throw this exception from find methods in `MediaMetaDataService` or delete the exception altogether from tests as well.